src/models/train_model.py

```diff
@@ -20,21 +31,20 @@ def train(epochs=20, lr=0.001):
     for e in range(epochs):
         running_loss = 0
         for images, labels in train_set:
-
+            
             optimizer.zero_grad()
-
+        
             log_ps = model(images)
             loss = criterion(log_ps, labels)
             loss.backward()
             optimizer.step()
-
+        
             running_loss += loss.item()
 
         print(f"Epoch: {e} - Training loss: {running_loss/len(train_set):5f}")
         train_loss.append(running_loss / len(train_set))
-
-    torch.save(model.state_dict(), "models/trained_model.pt")
+    torch.save(model, "models/trained_model.pt")
```

Comment:
Isn't it better to save `model.state_dict()`?
[https://stackoverflow.com/questions/42703500/best-way-to-save-a-trained-model-in-pytorch](url)